frontend/src/app/Container.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
```

Comment:
Should: remove the React import, it isn't needed anymore

Reply:
How comes it isn’t needed?

Reply:
Here's an explanation: https://www.chakshunyu.com/blog/why-did-and-dont-you-need-to-import-react/ also here in the docs: https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html

Reply:
Ah, I should’ve have rephrased that as what do I replace the React.Node with  as the child component? But I get that otherwise importing React isn’t necessary

Reply:
My issue fixed having to import react 🎉